Parser/AbstractParser.php

```diff
@@ -42,6 +43,12 @@ abstract class AbstractParser
      */
     protected $userAgent;
 
+    /**
+     * Holds the user agent the should be parsed
+     * @var ClientHints
+     */
+    protected $clientHits;
```

Comment:
Shouldn't this be `protected $clientHints;`?

Reply:
sure. will change that